### PR TITLE
add ability to read root directory

### PR DESCRIPTION
**Pull request by shamilovstas**
Added ability to go to root directory("/"). This ability may be needed by users of this library(in this case, me)

### CODE REVIEW

File: android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java [hunk 1]
```diff
@@ -309,7 +308,7 @@ public void onClick(View view) {
         int id = view.getId();
         if (id == R.id.previous_dir_imagebutton) {
             File parent = new File(currentDirectoryPath).getParentFile();
-            if (parent != null && parent.canRead()) {
+            if (parent != null) {
```
Comment (MostafaNasiri) [inline android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java:311]: Have you tested it on Android 6?

Comment (shamilovstas) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java:311]: Yes, I'm using Xiaomi Redmi 3S with Android 6.0.1

Comment (MostafaNasiri) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java:311]: I think there is no point in doing this. Doing a null check and checking if the app can access a directory are the same in this situation. In your code, if the app is not allowed to access a directory then listFiles method returns null and nothing will be shown to the user. My code achieves the same result. If a directory is readable then its files will be listed and if it's not then the code will not try to list it's files.

Comment (shamilovstas) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java:311]: I don't agree with you. In my project I need to go to root directory. My changes allow me to do that. In your variant my maximum of directory hierarchy is the "storage/emulated/0"

Comment (MostafaNasiri) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java:311]: Ok I'll check and change it if necessary. Version 1.2 will be published soon.

**Pull request closed without merging**